.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish release
+on:
+  release:
+    types: [published]
+permissions:
+  contents: read
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [lts/*]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build the `pdfjs-dist` library
+        run: npx gulp dist
+
+      - name: Publish the `pdfjs-dist` library to NPM
+        run: npm publish ./build/dist
```

Comment:
💡 Now that we have GitHub Actions-based publish, would you mind adding [Provenance statements](https://docs.npmjs.com/generating-provenance-statements)? It's literally 3 lines of code :)

Reply:
Actually, what the hell, it's open source. Let me raise a PR :joy: